guide/src/templates/template_defined_placeholders.md

```diff
@@ -40,19 +40,18 @@ Consider the following `cargo-generate.toml` file:
 ```toml
 [placeholders]
 project_name = { prompt = "Enter project name", default = "my_project", type = "string" }
-use_git = { prompt = "Initialize Git repository?", choices = ["Yes", "No"], default = "Yes", type = "boolean" }
+use_git = { prompt = "Initialize Git repository?", default = true, type = "bool" }
```

Comment:
logically you're making sense here. However, the intent was to showcase the `choices`. 
So the better fix would be `type = "string"`

But maybe we keep this, and add a different example for `choices`.

Reply:
Ah, makes sense. I added a new example showcasing the choices feature:  https://github.com/cargo-generate/cargo-generate/pull/1191/commits/887df5f64d3a8e8f84f3ac5028b115773604fb70